api/src/main/java/marquez/db/migrations/V43__UpdateRunsWithJobUUID.java

```diff
@@ -0,0 +1,56 @@
+package marquez.db.migrations;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import lombok.extern.slf4j.Slf4j;
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+
+/**
+ * This migration is dependent on the migration found in the SQL script for V42. This updates the
+ * runs table to include the <code>job_uuid</code> value for each record. We update the table in
+ * batches to avoid table-level locks so that concurrent reads and writes can continue to take
+ * place. Auto-commit is enabled, so it is entirely possible that this migration will fail partway
+ * through and some records will retain the <code>job_uuid</code> value while others will not. This
+ * is intentional as no harm will come from leaving these values in place in case of rollback.
+ */
+@Slf4j
+public class V43__UpdateRunsWithJobUUID extends BaseJavaMigration {
```

Comment:
TBH, I wrote it in Java because I wanted to be able to loop through the results and update a batch of run records at a time, rather than locking the whole table for writes. I _could_ do this in plpgsql, but I don't have much experience writing those kinds of looping functions, so I went with my comfort zone 😀. I can take a stab at doing this in pure SQL...

Re: the ordering, I could rename the Java class to be `V42a...` or some other suffix so that it comes alphanumerically after the `V42` SQL migration, but allow new migration scripts to continue with `V43__`, etc. But I'll leave that as a last resort if the pure SQL function doesn't pan out